src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -1779,6 +1779,15 @@
 short-summary: Manage DNS domains in Azure.
 """
 
+helps['network dns list-references'] = """
+type: command
+short-summary: Returns the DNS records referencing the target resource IDs.
+examples:
+  - name: List DNS records referencing the target resource IDs.
+    text: >
+        az network dns list-references --parameters /subscriptions/**921/resourceGroups/MyRg/providers/Microsoft.Network/trafficManagerProfiles/MyTm
+"""
+
```

Comment:
Above example is better to be added in the `Example` section at the bottom of workspace UI.

Reply:
Added example in AAZ tool for registered commands

Reply:
So, let's remove it in _help.py.